code/obj/item/device/pda2/bot_control.dm

```diff
@@ -159,9 +159,13 @@
 				else
 					src.lockdown = 0
 				var/area/guardthis = input(usr, "Please type 'Here' or the name of an area. Capitalization matters!", "GuardTron 0.0.1a", "Here") as text
-				if((BOUNDS_DIST(get_turf(usr), get_turf(src.master)) == 0))
+				var/turf/summon_area = get_turf(usr) // summon_turf is already used
```

Comment:
don't name it `area`, since that is a completely different thing

Reply:
Would summon_location work?

Reply:
Ok, switched it to use summone_location rather than summon_area